package.json

```diff
@@ -6,9 +6,9 @@
   "module": "index.ts",
   "scripts": {
     "build": "skyux build-public-library",
-    "test": "skyux test",
-    "test:watch": "skyux watch",
-    "start": "skyux serve",
+    "test": "skyux test --coverage library",
+    "test:watch": "skyux watch --coverage library",
+    "start": "skyux serve --hmr",
```

Comment:
Not necessarily opposed, but this would introduce different "default" (if they're just running `npm start) than what a SPA has.  https://github.com/blackbaud/skyux-sdk-template/blob/master/package.json#L10